src/main/java/org/folio/dataexp/service/DataExportTenantService.java

```diff
@@ -57,6 +70,13 @@ public void loadReferenceData() {
     loadConfiguration();
   }
 
+  @Override
+  public synchronized void createOrUpdateTenant(TenantAttributes tenantAttributes) {
+      var tenantId = super.context.getTenantId();
+      var tenant = StringUtils.isEmpty(tenantId) ? myUniversity : tenantId;
+      System.setProperty(VIEWS_VARIABLE_FOR_SQL_SCRIPT, tenant);
```

Comment:
it should be always taken from request headers, no reason to use value defined outside

Reply:
fixed